helm/aws-load-balancer-controller/templates/webhook.yaml

```diff
@@ -181,7 +181,7 @@ webhooks:
       name: {{ template "aws-load-balancer-controller.webhookService" . }}
       namespace: {{ $.Release.Namespace }}
       path: /validate-networking-v1-ingress
-  failurePolicy: Fail
+  failurePolicy: {{ .Values.webhookFailurePolicy | default "Fail" }}
```

Comment:
can we maintain the default value `Fail` in the values.yaml. Same for all the change lines on failurePolicy above